R/options.r

```diff
@@ -1,7 +1,5 @@
 #' @usage
-#' options(jupyter.* = ...)
-#' getOption('jupyter.*')
-#' jupyter_option_defaults$jupyter.*
+#' jupyter_option_defaults
```

Comment:
always sad when automated tests make things actually *worse*.

we should maybe just pick one option that we set and get as an example instead of removing the examples.